compiler/rustc_codegen_gcc/src/lib.rs

```diff
@@ -357,7 +357,7 @@ impl Deref for SyncContext {
 
 unsafe impl Send for SyncContext {}
 // FIXME(antoyo): that shouldn't be Sync. Parallel compilation is currently disabled with "-Zno-parallel-llvm".
-// TODO: disable it here by returing false in CodegenBackend::supports_parallel().
+// TODO: disable it here by returning false in CodegenBackend::supports_parallel().
```

Comment:
rustc_codegen_gcc changes should be reverted and put in their own pr against that repository

Reply:
Reverted